src/devices/wirenboard.ts

```diff
@@ -377,11 +377,14 @@ const definitions: Definition[] = [
             // buzzer
             await device.getEndpoint(4).read('genOnOff', ['onOff']);
 
+            // disable internal blinking zigbee state green led on start
+            await device.getEndpoint(5).write('genBinaryOutput', {0x0055: {value: 0x00, type: 0x10}});
+
             device.powerSource = 'Mains (single phase)';
             device.save();
         },
         endpoint: (device) => {
-            return {'default': 1, 'l1': 2, 'l2': 3, 'l3': 4};
+            return {'default': 1, 'l1': 2, 'l2': 3, 'l3': 4, 'l4': 5};
```

Comment:
Is `l4` also a switch? Then `e.switch().withEndpoint('l4'),` needs to be added.

Reply:
I've tried it: toggle showed up in Z2M UI, but it didn't actually switch led on/off.